app/src/main/java/pl/tlasica/firewire/play/Game.java

```diff
@@ -122,8 +124,15 @@ public void restart() {
 
     public void create(SurfaceHolder surfaceHolder, Queue<MouseEvent> eventsQueue) {
         Log.i("Game", "create()");
-        gameLoop = new GameLoop(this, surfaceHolder, eventsQueue);
-        thread = new Thread(gameLoop);
+        if (gameLoop == null) {
+            Log.i("Game", "gameLoop was null -> starting.");
+            gameLoop = new GameLoop(this, surfaceHolder, eventsQueue);
+            thread = new Thread(gameLoop);
+        }
+        else {
+            Log.i("Game", "gameLoop was not null -> updating");
```

Comment:
Seems it is not required to update as i works as expected after switching to different window